docs/remote-development/aws-cloud9.md

```diff
@@ -236,6 +192,30 @@ Filesystem      Size  Used Avail Use% Mounted on
 /dev/xvda1       20G  9.3G   11G  48% /
 ```
 
+## Install Docker-Compose
+
+By default, Docker is installed on AWS EC2's Ubuntu but Docker-Compose is not, so we have to install it ourselves.
+
+1. Run to download the current stable version of Docker-Compose:
+
+```
+sudo curl -L "https://github.com/docker/compose/releases/download/1.29.2/docker-compose-$(uname -s)-$(uname -m)" -o /usr/local/bin/docker-compose
```

Comment:
One thing that's happened recently is that they added `compose` to `docker` as a command vs. needing to use `docker-compose`.  Can we leverage that here?

Reply:
INTERESTING I did not know this. I'm able to run `docker compose --env-file config/env.development up -d` locally and it works as expected. I guess this warrants an update to [environment-setup.md](https://github.com/Seneca-CDOT/telescope/blob/master/docs/environment-setup.md#docker-and-docker-compose-setup) now because it also includes Docker-Compose installation instructions.

Reply:
Okay so `docker compose --env-file .env up -d` does not work in Cloud9 for some reason. When I spin up a new C9 environment, it installs the latest version of Docker so I assumed `docker compose` would work.

When I run `docker version`, I get
```
cindyle:~/environment/telescope (master) $ docker version
Client: Docker Engine - Community
 Version:           20.10.10
 API version:       1.41
 Go version:        go1.16.9
 Git commit:        b485636
 Built:             Mon Oct 25 07:42:57 2021
 OS/Arch:           linux/amd64
 Context:           default
 Experimental:      true

Server: Docker Engine - Community
 Engine:
  Version:          20.10.10
  API version:      1.41 (minimum version 1.12)
  Go version:       go1.16.9
  Git commit:       e2f740d
  Built:            Mon Oct 25 07:41:06 2021
  OS/Arch:          linux/amd64
  Experimental:     false
 containerd:
  Version:          1.4.11
  GitCommit:        5b46e404f6b9f661a205e28d59c982d3634148f8
 runc:
  Version:          1.0.2
  GitCommit:        v1.0.2-0-g52b36a2
 docker-init:
  Version:          0.19.0
  GitCommit:        de40ad0
```

Reply:
According to [this](https://github.com/docker/compose-cli/issues/1820), `docker compose` command is only available via Docker Desktop